src/scene/materials/standard-material.js

```diff
@@ -220,6 +220,7 @@ let _params = new Set();
  * indices of refraction, the one around the object and the one of its own surface. In most
  * situations outer medium is air, so outerIor will be approximately 1. Then you only need to do
  * (1.0 / surfaceIor).
+ * @property {number} f0 Defines the f0 weight as calculated using ((ior - 1) / (ior + 1))^2
```

Comment:
Just wondering - does this really need to be public API? Would we expect somebody to set this in code considering it seems to be directly derived from the `ior (refractionIndex)` value?